csFastFloat/Constants/Constants.cs

```diff
@@ -19,7 +22,32 @@ internal static class Constants
     internal const int smallest_power_of_five = -342;
     internal const int largest_power_of_five = 308;
 
-    internal readonly static ulong[] power_of_five_128 = {
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static byte get_powers(uint n)
+    {
+      Debug.Assert(n < powersTable.Length);
+      ref byte tableRef = ref MemoryMarshal.GetReference(powersTable);
+      return Unsafe.AddByteOffset(ref tableRef, (IntPtr)n);
+    }
+
+    private static ReadOnlySpan<byte> powersTable => new byte[19] {
```

Comment:
Note for `new byte[19]`: the count isn't necessary, but sometime I made the mistake to enter to less / to much elements in the array and the compiler won't complain. With the count the compiler complains if the element count doesn't match. So it's a little helper.